ui/app/templates/components/shamir-modal-flow.hbs

```diff
@@ -131,24 +146,24 @@
           {{/if}}
           <div class="box is-shadowless is-marginless no-padding-top is-fullwidth no-padding-sides" data-test-form-text>
             {{#if otp}}
-              <div>
-                <h4 class="title is-7 has-bottom-padding-m is-fullwidth">
+              <div class="has-bottom-margin-xxl">
+                <h4 class="field-title">
                   One-time password (OTP)
                 </h4>
+                <p class="help has-text-grey has-bottom-margin-xs">
+                  This OTP will be used to encode the generated operation token. <span class="has-text-semibold">Save this</span>, as you will need it later to decode the operation token.
+                </p>
                 <div class="message is-list has-copy-button" tabindex="-1">
                   <HoverCopyButton @copyValue={{otp}} />
                   <code class="is-word-break">{{otp}}</code>
                 </div>
               </div>
-              <div class="has-text-grey is-size-8 has-bottom-margin-l">
-                This OTP will be used to encode the generated operation token. <span class="has-text-semibold">Save this</span>, as you will need it later to decode the operation token.
-              </div>
             {{/if}}
             <p>
               Generate an operation token by entering a portion of the master key. Once all portions are entered, the generated token may be used to manage your secondary Disaster Recovery cluster.
```

Comment:
nit: we should specify that this needs to be a portion of the primary's master key.